layers/+tools/lsp/README.org

```diff
@@ -320,6 +321,24 @@ whereas with ~lsp-navigation~ set to ~'peek~, this is equivalent to:
 
 etc.
 
+** Bind to lsp upstreams
+Alternative to the binding shemes above. If you set ~lsp-use-upstream-bindings~
```

Comment:
- shemes -> schemes
- This is not a complete sentence. For example, "Alternatively, you can use vanilla LSP binding scheme, by setting ~lsp-use-upstream-bindings~ to ~t~."

Reply:
typo needs to be fixed